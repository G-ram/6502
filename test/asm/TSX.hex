a2 04 9a a2 05 ba 
