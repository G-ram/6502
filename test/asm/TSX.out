A: 00
X: 04
Y: 00
P: 00010000
S: 04
PC: 606
100 : 00
101 : 00
